projects/txm.json

```diff
@@ -12,14 +12,14 @@
   "desc_en": "The core of the tool is an algorithm that finds the optimal set of transplantations among multiple incompatible donor-recipient pairs. The optimization is performed with respect to the graft survival potential but also to the total expected life length and quality of each recipient. The tool is in production use by the Institute of Clinical and Experimental Medicine in Prague in an international program that manages kidney exchanges between the Czech Republic, Austria and Israel.",
   "desc_de": "Der Kern dieses Tools ist ein Algorithmus, der die optimale Kombination von Transplantationen unter mehreren inkompatiblen Spender-Empfänger-Paaren findet. Die Optimierung berücksichtigt das Überlebenspotenzial des Transplantats, aber auch die erwartete Gesamtlebensdauer und -qualität jedes Empfängers. Das Tool wird vom Institut für klinische und experimentelle Medizin in Prag in einem internationalen Programm zur Verwaltung des Nierenaustauschs zwischen der Tschechischen Republik, Österreich und Israel benutzt.",
   "desc_pieces_cs": [
-    "TX Matching je algoritmus, který z množiny několika desítek dvojic nekompatibilních dárců a příjemců ledvin dokáže vybrat optimální kombinaci transplantací.",
-    "Někteří pacienti, kteří potřebují novou ledvinu, mají to štěstí, že ve svém okolí najdou někoho, kdo je ochoten jim ledvinu darovat. Většinou to bývá příbuzný nebo blízký přítel, ale ani to nezaručuje, že bude dárce kompatibilní. S využitím TX Matching algoritmu je možné mezi velkým množstvím takovýchto dvojic najít řetězce vhodných transplantací a dát tak naději na běžný život více pacientům.",
-    "Algoritmus bere v úvahu vhodnost transplantačního štěpu, naději na dožití i celkovou kvalitu života příjemce. Již dnes je TX Matching využívám pražským IKEM ke koordinaci mezinárodního programu výměny ledvin mezi Českou republikou, Izraelem a Rakouskem."
+    "Párová výměna ledvin je způsob, jak zvýšit počet možných transplantací ledvin a zvýšit tím délku a kvalitu života pacientů s chronickým onemocněním ledvin. V párové výměně dárce daruje ledvinu jinému příjemci a na oplátku získá kompatibilní ledvinu pro svého rodinného příslušníka nebo blízkého přítele.",
+    "TX Matching je nástroj, který umožňuje párovou výměnu zorganizovat a najít v rámci ní optimální kombinaci dárců a příjemců. Organizátor párové výměny využije na její manažment kontrolu uživatelského přístupu, automatické vkládání anonymizovaných pacentských dat přes rozhraní API a možnost reportingu. Algoritmus hledání optimálního párování obsahuje množství nastavitelných omezení dle potřeb dané organizace.",
+    "Algoritmus TX Matchingu je postaven na otevřeném kódu a jeho aktuální verze je vždy přístupna k využití zdarma. Aktuálně je program TX Matching aktivně využíván pražským IKEM na manažment a optimalizaci mezinárodního programu párové výměny mezi Českou republikou, Izraelem a Rakouskem."
```

Comment:
na open source kódu (je to dostatečně česky, bohužel) a celá platforma je volně dostupná odborné i laické veřejnosti... IKEM na správu a optimalizaci...

Reply:
laické veřejnosit?